### DISCUSSION TITLE
You have sent too many messages to the model. Please try again later

### DISCUSSION THREAD

**Discussion by luluhuhu** [acheong08/ChatGPT-Proxy-V4#88] [Q&A] [answered]
The following error was reported:

OpenAI: {"detail": {"message": "You have sent too many messages to the model. Please try again later.", "code": "model_cap_exceeded", "clears_in": 5016}

May I ask what the reason is?

**Comment by gptlang** [comment]
"You have sent too many messages to the model. Please try again later."

TL;DR Rate limited

**Reply by luluhuhu** [reply]
Thank you for your reply.

Is it an error returned by OpenAi?

Or is it an error returned by ChatGPT-Proxy-V4?

**Reply by luluhuhu** [reply]
TL; DR

What is the speed?

Thank you

**Reply by acheong08** [reply, marked as answer]
Obviously OpenAI. I have no reason to limit anything

The speed is 50 req per 1 hour per account
